src/services/refactors/addOrRemoveBracesToArrowFunction.ts

```diff
@@ -70,10 +70,12 @@ namespace ts.refactor.addOrRemoveBracesToArrowFunction {
         return { renameFilename: undefined, renameLocation: undefined, edits };
     }
 
-    function getConvertibleArrowFunctionAtPosition(file: SourceFile, startPosition: number): Info | undefined {
+    function getConvertibleArrowFunctionAtPosition(file: SourceFile, startPosition: number, userRequested = true): Info | undefined {

```

Comment:
I can see the convenience of having this be called `userRequested` in all the different refactorings, but, personally, I think it would be clearer to give it a name that's specific to each function you've added it to.  For example, in this case, the parameter could be called something like `considerFunctionBodies` - that would be more meaningful both within the function and at the call-sites that aren't part of a flow that can be either implicit or user-requested.  As a bonus, it would also address my comment above, that this parameter defaults to explicit, whereas the ones in the protocol default to implicit.